docs/api.md

```diff
@@ -153,6 +160,19 @@ Renews the token if, for instance, new permissions are required.
 
 **Kind**: instance method of [<code>CozyClient</code>](#module_CozyClient)  
 **Returns**: <code>object</code> - Contains the fetched token and the client information.  
+<a name="module_CozyClient+setData"></a>
+
+### cozyClient.setData(data)
```

Comment:
I missed the PR that introduced this method, but it's a bad idea imho: its name is too generic and doesn't reflect its use. You can be sure that it will be used for bad things. And moreover, I don't get in which cases the PouchDB link needs to update the store itself. What's the precise use case?

Reply:
Yeah I saw it (late), but it doesn't add much info

Reply:
We need to update the store to notify a Query component that it's query has been updated after a pouch replication completed. Maybe we can do it in another way ?